packages/manager/modules/pci/src/projects/project/additional-ips/order/add.html

```diff
@@ -122,6 +124,7 @@
                     data-translate_values="{
                         guideLink: $ctrl.getAdditionalIpConfGuideLink()
                     }"
+                    data-ng-click="$ctrl.trackConfigGuideClick($event)"
```

Comment:
**question:** Why are we using a `ngClick` directive on a `p` element?
Couldn't we use a semantic element instead?

Reply:
Hello @antleblanc ,
Here we display the link to the guide in the `oui-message` banner. And the `<a>` tag is added as part of translation value. And we need to track the click on this `<a>` tag. But since the tag is part of translation, can't really have a callback triggered in the controller. So I have implemented a listener on the parent and then filtering the events to know if anchor tag is clicked.

Besides, this implementation does not affect the accessibility.

Reply:
What if I'm using keyboard to navigate through elements?

Reply:
As discussed together, we'll remove this `ngClick` directive on `<p>` element.

Reply:
Hello @antleblanc, the suggested way using `track-click` directive in the translation file does not trigger the track.

Reply:
Hello @antleblanc , In this staging environment https://feat-additional-ips-tracking-manager.eu.dtci.ovhcloud.dev/container/#/public-cloud/pci/projects/2ebe3ff71ad242609110221d21a33338/additional-ips/order, we can observe that the proposed solution does not track the click of guide.

Steps to verify,

1. Navigate to URL https://feat-additional-ips-tracking-manager.eu.dtci.ovhcloud.dev/container/#/public-cloud/pci/projects/2ebe3ff71ad242609110221d21a33338/additional-ips/order
2. Select Fail-over Ip in the first step
3. Then select any region
4. In the step 3, click on the link "plus de détails ici", we can notice that this click is not tracked.

Could you please tell me how to proceed?

Reply:
The suggested way of implementation works. So `ng-click` on `<p>` elements has been removed.